package.json

```diff
@@ -39,7 +40,8 @@
     "semver": "^5.3.0",
     "server-destroy": "1.0.1",
     "tcomb": "^3.0.0",
-    "tcomb-validation": "^3.0.0"
+    "tcomb-validation": "^3.0.0",
+    "twin-bcrypt": "^2.1.1"
```

Comment:
I should use [`bcrpyt`](https://github.com/kelektiv/node.bcrypt.js) by default, and switch to `twin-bcrypt` in specific projets which don't support C++ modules.